target/linux/ar71xx/image/tp-link.mk

```diff
@@ -510,6 +510,21 @@ define Device/tl-wa850re-v2
   MTDPARTS := spi0.0:128k(u-boot)ro,1344k(kernel),2304k(rootfs),256k(config)ro,64k(art)ro,3648k@0x20000(firmware)
 endef
 
+define Device/tl-wa855re-v1
```

Comment:
Could you please make here something like `Device/tl-wa85xre`, put it above `Device/tl-wa850re-v2`, include there common variables and then call it inside `Device/tl-wa850re-v2` and `Device/tl-wa855re-v1` with `$(Device/tl-wa85xre)`. Both models (TL-WA850RE v2 and TL-WA855RE v1) are very similar and share most of the variables. As an example, please take a look at `Device/archer-cxx` and `Device/archer-c25-v1`, `Device/archer-c59-v1` and `Device/archer-c60-v1`.